sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/ChangeFeedProcessorItem.java

```diff
@@ -55,8 +60,18 @@ public JsonNode getPrevious() {
      * @return change feed metadata.
      */
     @Beta(value = Beta.SinceVersion.V4_37_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    @JsonIgnore
     public ChangeFeedMetaData getChangeFeedMetaData() {
-        return changeFeedMetaData;
+
+        if (this.changeFeedProcessorItemAsJsonNode == null) {
```

Comment:
why we need to construct changeFeedProcessorItemAsJsonNode? does not seem to be used in this method

Reply:
Fixed in next iteration.